.github/workflows/builds-review.yaml

```diff
@@ -0,0 +1,62 @@
+name: Review builds
+
+on:
+  pull_request:
+    types:
+      - labeled
+
+jobs:
+  build:
+    if: |
+      github.event_name == 'pull_request' &&
+      github.event.label.name == 'build::review'
+    strategy:
+      matrix:
+        include:
+          - runner: ubuntu-latest
+            subdir: linux-64
+          - runner: macos-latest
+            subdir: osx-64
+          - runner: windows-latest
+            subdir: win-64
+    runs-on: ${{ matrix.runner }}
+    steps:
+      - name: Remove build label
+        uses: actions/github-script@v5
+        with:
+          github-token: ${{ secrets.CANARY_ACTION_TOKEN }}
+          script: |
+            const { data: pullRequest } = await github.rest.pulls.get({
+              ...context.repo,
+              pull_number: context.issue.number,
+            })
+            const buildLabel = '${{ github.event.label.name }}'
+            const labels = pullRequest.labels.map(label => label.name)
+            const hasBuildLabel = labels.includes(buildLabel)
+
+            if (hasBuildLabel) {
+                await github.rest.issues.removeLabel({
+                  ...context.repo,
+                  issue_number: context.issue.number,
+                  name: buildLabel,
+                })
+            }
+
+      # Clean checkout of specific git ref needed for package metadata version
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.ref }}
+          clean: true
+          fetch-depth: 0
+
+      - name: Create and upload review build
+        uses: conda/actions/canary-release@canary-release
```

Comment:
We need to remember to update this after merging https://github.com/conda/actions/pull/10